jscomp/syntax/ppx_entry.ml

```diff
@@ -255,8 +255,26 @@ let handle_core_type
   | _ -> super.typ self ty
 
 
-
-
+(*A very simple check if the string is a valid js regex*)
+let js_regex_checker s = 
+  let l = String.length s in
+  String.get s 0 = '/' && (String.get s (l-1) = '/' || String.get s (l-2) = '/')
+
```

Comment:
How do you run the `ounit` tests? I think `make test` tests the js files in tests directory. Thanks!